api/swagger/swaggerDoc.json

```diff
@@ -633,66 +630,6 @@
       }
     },
     "400ErrorResponseV02": {
-      "required": [
```

Comment:
Is specifying too much detail on the error responses a no-no?

Reply:
Kevin suggested removing `required` on any definitions related to responses, so that is taken care of in https://github.com/NYPL/dgx-patron-creator-service/pull/14

Reply:
`required` is really intended more for the Request object (i.e. what parameters or fields are required) so it's a little strange (and maybe even misleading/inaccurate) to have it in the Response definition. That's why I suggested removing it.